src/extensionApi.ts

```diff
@@ -199,8 +199,12 @@ export class CommandHandler {
 
     async createServer(): Promise<Protocol.Status> {
         this.assertServersDataExists();
-        const download: string = await vscode.window.showQuickPick(['Yes, please download', 'No, use runtime on my disk'],
+        const download: string = await vscode.window.showQuickPick(
+            ['Yes, please download', 'No, use runtime on my disk'],
```

Comment:
Would be good to have shorter answers: IMO: 
* `Download runtime`
* `Use runtime on my disk`

Reply:
@dgolovin I dont think that your shorter answers are better: having Yes/No in the beginning of the text makes it completely obvious what the result is. The rest of the sentence then only provides details